agent/result_parser.py

```diff
@@ -43,9 +45,7 @@ def parse_results(results: whois.parser.WhoisCom) -> Iterator[Dict[str, Any]]:
                 ),
                 "name": name,
                 "emails": get_list_from_string(
-                    scan_output_dict.get("email", "")
-                    if scan_output_dict.get("email", "") != ""
-                    else scan_output_dict.get("emails", "")
+                    scan_output_dict.get("email") or scan_output_dict.get("emails", "")
```

Comment:
To be more generic, I think validating the email address is better.

Reply:
It's already included in whois codee here : https://github.com/GroupSense/whois/blob/011b6d0bd5b62fc30e18bf137e3e5270a586a502/whois/parser.py#L33

but for .at domains where there is only one email. there is no check.

I added a extra check in our code for better conrol.